documentation/docs/strategies/advanced-mm/hanging-orders.md

```diff
@@ -1,16 +1,28 @@
 # Hanging Orders
 
-This feature keeps orders "hanging" (or not cancelled and remaining on the order book) if a matching order has been filled on the other side of the order book.
+**Updated as of `v0.28.0`**
+
+This feature keeps orders "hanging" (or not cancelled and remaining on the order book) if a matching order has been filled on the other side of the order book (bid vs. ask order books).
 
 ## How It Works
 
-Typically, orders are placed as pairs in single order mode (1 buy and 1 sell order). The parameter `hanging_orders_enabled` allows Hummingbot to leave the order on the other side hanging (not cancelled) whenever one side is filled.
+Typically, orders are placed as pairs in single order mode (1 buy and 1 sell order), and when a buy or sell order is filled, the other order is cancelled. The parameter `hanging_orders_enabled` allows Hummingbot to leave the order on the other side hanging (not cancelled) whenever one side is filled.
 
 The hanging order will be cancelled in the following conditions:
 
 1. The spread goes above the specified `hanging_orders_cancel_pct` value
 2. Sending `stop` or `exit` command
 
+Type `config hanging_orders_enabled` and `config hanging_orders_cancel_pct` to set values for these parameters. 
+
+## Illustrative Example
```

Comment:
For the example in hanging orders, I recommend illustrating how hanging orders works over a series of price movements, since this is how users use it in practice. For example, let's say the price of ETH rises from 200 to 210 over 5 steps, and then goes down to 200 in another 5 steps. A user who has hanging orders on will end up selling in each step on the way up, and then the hanging orders will trigger on the way down. A table or chart might be helpful to illustrate this.